example/REQUIRE

```diff
@@ -0,0 +1 @@
+ConjugatePriors
```

Comment:
It is not needed for the tests. This is just an informal way to state that the examples have additional requirements on top of REQUIRE. We should put this into the readme too.